packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -51,6 +52,10 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
     };
   }
 
+  public addAdditionalToolbarItems(toolbarItems: ToolItemDef[]): void {
```

Comment:
why wouldnt you just add additional tool bar items to MeasureToolsUiProviderOptions?

Reply:
Yeah, I'd put it as an option to the ctor. "Add" also here is really a replace, which I think consumers would not expect they'd be overwriting someone else's addition.

Reply:
Moved additionalToolbarItems to MeasureToolsUiProviderOptions